src/main/java/appeng/core/localization/GuiColors.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * This file is part of Applied Energistics 2.
+ * Copyright (c) 2013 - 2014, AlgorithmX2, All rights reserved.
+ *
+ * Applied Energistics 2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Applied Energistics 2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Applied Energistics 2.  If not, see <http://www.gnu.org/licenses/lgpl>.
+ */
+
+package appeng.core.localization;
+
+import appeng.core.AELog;
+import net.minecraft.util.StatCollector;
+
+public enum GuiColors
+{	
+	//ARGB Colors: Name and default value
+	SearchboxFocused			(0x6E000000),
```

Comment:
You seem to be mixing tabs and spaces here

Reply:
Alright. Gonna delint when I get home from work.

Reply:
Are you sure that I'm mixing tabs and spaces? I checked the file thoroughly and it seems to be a consistent use of tabs throughout the file.

I used tab size 4 when coding. You want me to adjust to a different size?

Reply:
Looks like github uses a different tab size, but that doesn't matter much